static/app/components/sentryReplayInit.tsx

```diff
@@ -0,0 +1,34 @@
+import {useEffect} from 'react';
+
+import useOrganization from 'sentry/utils/useOrganization';
+
+/**
+ * Load the Sentry Replay integration based on the feature flag.
+ */
+export function SentryReplayInit() {
```

Comment:
Looks like this should actually be a hook, but guessing this is probably used in some old class components?